src/features/featureQuery.js

```diff
@@ -357,7 +357,7 @@ FeatureQuery.prototype.lookupFeature = function (query) {
         tag: query.tag, script: query.script
     });
     if (!feature) return new Error(
-        `font '${this.font.names.fullName.en}' ` +
+        `font '${(this.font.names.unicode || this.font.names.macintosh).fullName.en}' ` +
```

Comment:
This is because the Macintosh name will always exist whereas the windows one is specific to windows only, i can change it to ``(this.font.names.unicode || this.font.names.macintosh|| this.font.names.windows).fullName.en`` if you think that's better though.

Reply:
or actually... hmm... i could do unicode, windows, macintosh.